Examples/PcapPlusPlus-benchmark/CMakeLists.txt

```diff
@@ -1,12 +1,39 @@
-add_executable(benchmark benchmark.cpp)
+add_executable(BenchmarkExample benchmark.cpp)
 
-target_link_libraries(benchmark PUBLIC PcapPlusPlus::Pcap++)
+target_link_libraries(BenchmarkExample PUBLIC PcapPlusPlus::Pcap++)
 
-set_target_properties(benchmark PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${PCAPPP_BINARY_EXAMPLES_DIR}")
+set_target_properties(BenchmarkExample PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${PCAPPP_BINARY_EXAMPLES_DIR}")
+
+if(PCAPPP_USE_GOOGLEBENCHMARK)
```

Comment:
Two reasons actually,

- Are want Google Benchmark as a dependency always? Google benchmarks require a compiler to support C++14
- This is not really important I can fix it probably but for MacOS (cross-compiling cases x86 -> ARM or vice versa I can't remember now) there was an error, and I can't figure out why. But then noticed the README specifies the benchmarks only run in linux. If all platforms supported, we should also remove this mention from README

Reply:
1. I don't mind having Google Benchmark as a dependency of this specific example, especially because it downloads it automatically
2. Is it possible to avoid compiling this example if the complier doesn't support C++14? I'm not a CMake expert but I assume this is possible?
3. > Then noticed the README specifies the benchmarks only run in linux. If all platforms supported, we should also remove this mention from README

    I didn't see where it's mentioned... I managed to compile and run it on Windows with MSVC 🤔

Reply:
1. Agreed
2. It should be I'll take a look
3. Probably a forgotten mention https://github.com/seladb/PcapPlusPlus/blob/db905ad02e3112b134106d89c604a51ace595f76/Examples/PcapPlusPlus-benchmark/README.md?plain=1#L8

Reply:
> 3. Probably a forgotten mention 

I think it can be compiled on all platforms so we can drop this comment